client/modules/ftml-wasm/tests/ftml-original-tests.ts

```diff
@@ -1,12 +1,10 @@
-import * as fse from "fs-extra"
-import * as fs from "fs/promises"
-import { sync as globby } from "globby"
-import * as path from "path"
-import { performance } from "perf_hooks"
-import * as uvu from "uvu"
-import * as assert from "uvu/assert"
+import { assert } from "@esm-bundle/chai"
 import * as lib from "../src/index"
-import type { IPageInfo, IParseWarning, ISyntaxTree } from "../vendor/ftml"
+import type { IPageInfo } from "../vendor/ftml"
+
+const jsons = import.meta.globEager("/../ftml/test/*.json")
+const htmls = import.meta.globEager("/../ftml/test/*.html?raw")
+const txts = import.meta.globEager("/../ftml/test/*.txt?raw")
```

Comment:
What does it do ro these files without raw?

Reply:
Tries to load them as JavaScript. Due to old web shenanigans, the file type of referenced file in a URL technically doesn't matter, so Vite follows this and doesn't assume the content type of a file, unless you tell it to.